src/components/cards/card.css

```diff
@@ -0,0 +1,171 @@
+:root{
+    --card-badge:rgb(255, 57, 90);
+    --text-overlay:rgb(201, 200, 200);
+    --text-overlay-full: rgb(224, 224, 224);
+}
+.card{
+    position: relative;
+}
+
+.img-xxl{
+    height: 30rem;
+    width: 30rem;
+    object-fit: contain;
+}
+.img-xl{
+    height: 25rem;
+    width: 25rem;
+    object-fit: contain;
+}
+.img-lg{
+    height: 20rem;
+    width: 20rem;
+    object-fit: contain;
+}
+.img-md{
+    height: 15rem;
+    width: 15rem;
+    object-fit: contain;
+}
+.img-sm{
+    height: 12rem;
+    width: 12rem;
+    object-fit: contain;
+
+}
+.img-xsm{
+    height: 5rem;
+    width: 5rem;
+    object-fit: contain;
+
+}
```

Comment:
Remove the unnecessary spaces wherever applicable.

Otherwise, the code is really neat.
Descriptive classes are used.